extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1628,16 +1628,18 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
           return;
         }
         // Create amp-pixel and append to document to send impression.
-        this.win.document.body.appendChild(
-          createElementWithAttributes(
-            this.win.document,
-            'amp-pixel',
-            dict({
-              'src': url,
-              'referrerpolicy': scrubReferer ? 'no-referrer' : '',
-            })
-          )
-        );
+        this.getAmpDoc()
+          .getBody()
+          .appendChild(
+            createElementWithAttributes(
+              this.win.document,
```

Comment:
Is this.win.document still correct?

Reply:
Yes (see a similar usage above). The param is to get an object to call `createElement` on.